src/gui/transferlistfilterswidget.cpp

```diff
@@ -583,6 +584,24 @@ TransferListFiltersWidget::TransferListFiltersWidget(QWidget *parent, TransferLi
     toggleCategoryFilter(pref->getCategoryFilterState());
     frameLayout->addWidget(m_categoryFilterWidget);
 
+    QCheckBox *tagsLabel = new QCheckBox(tr("Tags"), this);
+    tagsLabel->setChecked(pref->getTagFilterState());
+    tagsLabel->setFont(font);
+    connect(tagsLabel, SIGNAL(toggled(bool)), SLOT(onTagFilterStateChanged(bool)));
```

Comment:
I figured consistent syntax should be used within a file. I went ahead and ported all the old code's calls to Qt5 as well so the file would be consistent.